src/annotation/index.ts

```diff
@@ -814,6 +855,28 @@ export const annotationTypeHandlers: Record<
       callback(annotation.pointA, false);
       callback(annotation.pointB, false);
     },
+    defaultProperties(
+      annotation: Line,
+      annotationLayerPositions: Float32Array<ArrayBufferLike>[],
+      scales: Float64Array,
+      units: readonly string[],
+    ) {
+      annotation;
+      const properties: AnnotationNumericPropertySpec[] = [];
+      const values: number[] = [];
+      const length = lineLength(annotationLayerPositions, scales, units);
+      if (length) {
+        properties.push({
+          type: "float32",
+          identifier: "Length",
+          default: 0,
+          description: "Length of the line annotation in nanometers",
```

Comment:
Should probably just say "length of the line annotation", with a type of string, and then the value can include the appropriate unit suffix.

Reply:
Hi Jeremy, getting back to this now, are you suggesting adding "string" as a type of `AnnotationNumericPropertySpec` or Adding a `AnnotationStringPropertySpec`. I thought there was no string currently for performance reasons.

Reply:
Oh, I forgot that string properties aren't supported for annotations --- only for segment properties.  They would be challenging to support within a shader, and some changes to the precomputed annotation encoding would be needed to support strings in that format, but there is no fundamental difficulty with supporting them solely for display in the "Selection details" panel.  However, I realize that with the `format` member you added to AnnotationNumericPropertySpec we can anyway accomplish the same thing without adding string annotation support.

Reply:
In that case, would you suggest skipping the "X meters + Y seconds" idea and instead only calculate length if all the non-zeros are the same unit?